.circleci/scripts/auto-rebase-branches.txt

```diff
@@ -0,0 +1,3 @@
+jz/rebase-custom1
```

Comment:
Can we add some real branches here now?

Reply:
jz/rebase-custom1 and jz/rebase-custom2 are real branches I created to test auto-rebasing. We can update them to customization branches once testing passes.